<a href="https://colab.research.google.com/github/malith001/ME421_Mechanical_Systems_Lab-A04/blob/main/Heat_Transfer/E20389_Heat_Transfer.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## Part 1: Propose a method to find out the feasibility of the available unit.  

1. First, collect all the given data such as number of containers, mass of milk, mass of aluminum container, specific heat capacities, initial temperature, final temperature, and cooling time.

2. Consider the worst condition by taking the highest initial milk temperature for the analysis.

3. stimate the total cooling load required to cool both the milk and the aluminum containers up to the required storage temperature.

4. Include additional practical heat gains such as heat entering through walls, door opening, and air leakage, and apply a safety factor.

5. Find the cooling capacity of the available Air Conditioning unit from the nameplate or laboratory data.

6. Compare the required cooling load with the available cooling capacity of the unit to check whether the unit is sufficient.

7. Check whether the unit can maintain the storage temperature until the next day by considering the maintenance cooling load.

8. Perform an experiment using the available unit with an equivalent test load and record the temperature variation with time.

9. Verify whether the unit can reduce the temperature to the required value within the given time and maintain it steadily.

10. Repeat the same procedure for 80 milk containers per day and identify any modifications required if the available unit is not sufficient.




**Known Data:**

---


1. Number of containers = 40

Per container:

*   Milk mass = 5 kg
*   Container mass = 2 kg (Aluminum)

---



2. Specific heats:


*   Milk: $C_{p,\text{milk}}$ = 4000J/kg⋅K
*   Aluminum: $C_{p,\text{Al}}$ = 921.1J/kg⋅K

---



3. Target cooling:

*   $T_{initial}$ = 35–40°C
*   $T_{final}$ = 15°C

---



4. Worst case used for design:

*   $T_{initial}$ = 40°C
*   ΔT= 40−15 =25K

---



5. Heat Removal per Container:

Milk cooling load,

<center>
$Q_{milk}$ = m$C_{p}$ ​ΔT

$Q_{milk}$ = 5×4000×25

$Q_{milk}$ ​= 500,000J
</center>


Container cooling load,
<center>
$Q_{Al}$ = 2×921.1×25

$Q_{Al}$ = 46,055J
</center>

Total per container,
<center>
$Q_{container, total}$ = 500,000J+ 46,055J =546,055J
</center>

---



Total Heat Removal (40 containers),
<center>
$Q_{total}$ = 40×546,055

$Q_{total}$ = 21,842,200J

$Q_{total}$ ≈ 21.84MJ
</center>
Required Cooling Rate,

*   Cooling time = 2 hours = 7200 s

<center>
$\dot{Q}$ ​= Q/t

$\dot{Q} = $$\frac{21.84 \times 10^{6}}{7200}$

$\dot{Q}$ ​≈ 3033W

$\dot{Q}$ ≈ 3.03kW
</center>


---


Minimum refrigeration capacity required ≈ 3.1 kW.

---



Applying Safety Factors,

Include:



* Milk loading disturbance
* Fan & lighting loads
* Air infiltration
* Heat gains from walls
* Door opening losses





Typical safety margin:
<center>
1.2–1.5
</center>

Assume 1.4:
<center>

$Q_{design​}$ = 3.03×1.4

$Q_{design​}$ ≈ 4.24kW

</center>

Design cooling capacity required ≈ 4.3 kW

---



**Comparing with AC Unit Capacity**

Obtain AC nameplate data:

*   Cooling capacity (kW or BTU/hr)

Conversion:
*   1kW = 3412BTU/hr

Feasibility Criteria:

| Condition | Interpretation |
|----------|----------|
| AC < 3 kW       | Cannot meet 2-hour cooling        |
| AC ≈ 3–4 kW     | Marginal       |
| AC > 4.3 kW   | Suitable       |      



---
Evaluating Temperature Maintenance Load

After reaching 15 °C:

Only heat gains must be offset:

Sources:

*   Transmission through walls
*   Air leakage
*   Door opening
*   Warm container addition (if any)

Method:

<center>
$Q_{maintenance​​}$ =UA( $T_{ambient​}$-$T_{room​​}$)
</center>

Where:
*   ΔT = Temperature difference
*   A = Surface area
*   U = Overall heat transfer coefficient

If:

$Q_{maintenance​​}$ < $AC_{capacity}$

Temperature can be maintained.
___




# **Part 2: Experiment with the cooling unit in the Heat Engine Laboratory and obtain the necessary data.**

In [1]:
!pip install PyPDF2


   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 232.6/232.6 kB 6.0 MB/s eta 0:00:00


In [3]:
from google.colab import files
uploaded = files.upload()

Saving E_20_389_Observations.pdf to E_20_389_Observations (1).pdf


In [5]:
from PyPDF2 import PdfReader

reader = PdfReader("E_20_389_Observations.pdf")

for i, page in enumerate(reader.pages, start=1):
    text = page.extract_text()
    print(f"Page {i}")
    print(text)

Page 1
OBSERVATION SHEET  
 
Table 1: Experimental Observation Sheet for Cooling Unit Performance Evaluation  (Gas Side   
Temperature Measurements)   
Time 
(min)  Air  
Flow  
Rate  
(kg/hr)  Room  
Return  
Air  
Temp  
T3  
(°C)  Condenser  
Inlet  
Temp T1  
(°C)  Condenser  
Outlet  
Temp T2  
(°C)  Evaporator  
Inlet Temp 
T4 (°C)  Evaporator  
Outlet  
Temp T5  
(°C)  Refrigerant  
Pressure  
Low Side  
(MPa)  Refrigerant  
Pressure  
High Side  
(MPa)  
0  30  31.3  44.6  32.7  10.2  11.2  0.3  1.8  
1  30  31.2  44.3  32.8  10  11.5  0.3  1.8  
2  30  31.4  44.7  32.4  10.1  10.5  0.3  1.8  
3  30  31  44.4  32.3  10.2  11  0.3  1.8  
4  30  31.2  44.5  32.2  9.7  11  0.3  1.8  
5  30  31  44.1  32  9.6  10.8  0.3  1.8  
6  30  31.3  44.3  31.9  9.5  11.5  0.3  1.8  
7  30  31.4  44.4  32.1  9.1  10.2  0.3  1.8  
8  30  31.3  44.4  31.8  9.6  10.6  0.3  1.8  
9  30  31.1  44.3  31.6  9.7  10.6  0.3  1.8  
10  30  31.1  44.1  31.5  9  10  0.3  1.8  
11  30  31.1  44.5  31.6  9

___


# **Part 3: Determine the developments required to be introduced for the unit if required to store 80 milk containers per day.**

## **Recalculating Cooling Load for 80 Containers**

---

Previously for 40 containers:

Milk per container = 5 kg

Container mass = 2 kg
___

For 80 containers:

$m_{milk}$ ​= 80×5 = 400 kg

$m_{Al}$ ​= 80×2 = 160 kg
___


Temperature Reduction

Worst case cooling:

40°C → 15°C

Δ𝑇=25𝐾

Heat Removal from Milk

<center>$Q_{milk}$​= m$C_p$​ΔT

$Q_{milk}$​= 400×4000×25

$Q_{milk}$​= 40,000,000 J

$Q_{milk}$​= 40 MJ </center>

Heat Removal from Aluminum Containers

<center>$Q_{Al}$ ​= 160×921.1×25

$Q_{Al}$ ​= 3,684,400 J

$Q_{Al}$ ​= 3.68 MJ </center>

Total Cooling Load

<center>$Q_{Total}$ ​= (40+3.68) MJ

$Q_{Total}$ ≈ 43.68 MJ</center>

 **Required Cooling Capacity**

---

Cooling must be completed within 2 hours:

t=7200 s

<center>$\dot{Q} = $$\frac{43.68 \times 10^{6}}{7200}$

$\dot{Q}$ ≈ 6067 W

$\dot{Q}$ ≈ 6.07 kW</center>

Add Safety Factor (1.3–1.5)

Assume 1.4:

6.07×1.4 = 8.5 kW

Required Cooling Capacity ≈ 8–9 kW

**Comparing with Existing Unit**

---

***From Part 2 observations:***

* Refrigerant: R22

* Mass flow rate: 30 kg/hr

* Suction pressure: 0.3 MPa

* Discharge pressure: 1.8 MPa

* Evaporator inlet/outlet refrigerant temperatures ≈ 9–11°C

* $\dot{m}$ = 30 kg/hr

* Convert to kg/s: $\dot{m}$ = 0.00833 kg/s

* Low side pressure = 0.3 MPa

* Evaporator refrigerant temperature ≈ 10°C

**For R22:**

**At 0.3 MPa:**

* Saturation temperature ≈ 0–2°C

* Latent heat $h_{fg}$ ≈ 210kJ/kg

* Since measured temperature ≈ 10°C, refrigerant is slightly superheated.

**from R22 tables at 0.3 MPa:**

* $h_{in}$ (after expansion, saturated mixture) ≈ 230 kJ/kg

* $h_{out}$ (slightly superheated vapor) ≈ 410 kJ/kg

So:

<center>Δh = $h_{out}$ - $h_{in}$

Δh≈410−230

Δℎ≈180𝑘𝐽/𝑘𝑔

Δh≈180kJ/kg</center>

Cooling Capacity Calculation

<center>$\dot{Q}_{evap}$ = $\dot{m}$ Δh

$\dot{Q}_{evap}$ = 0.00833×180,000

$\dot{Q}_{evap}$ = 1499 W

$\dot{Q}_{evap}$  ≈ 1.5 kW</center>

# **Interpretation**

---

<center>$\dot{Q}_{actual}$  ≈ 1.5 kW</center>
 **Comparing With Required Load**

---


**From earlier design calculation:**

For 40 containers:

* Required ≈ 4.2 kW

* Your unit provides ≈ 1.5 kW

Therefore:

# **This shows that the available unit capacity of 1.5 kW is far below the required 4.2 kW. Hence, it cannot handle even 40 containers. For 80 containers, with a required capacity of about 8.5 kW, the unit is completely unsuitable.**
